pkg/volume/storageos/storageos.go

```diff
@@ -347,6 +349,14 @@ func (b *storageosMounter) SetUp(fsGroup *int64) error {
 		b.volNamespace = b.podNamespace
 	}
 
+	globalPDPath := makeGlobalPDName(b.plugin.host, b.pvName, b.podNamespace, b.volName)
```

Comment:
How does access pattern of storageOS works on multiple pods using same volume on same node. Can the volume be attached multiple times on the node with different paths?  

In general I am not sure why this volume uses pod's namespace in something that it calls "GlobalPDPath". That implies this path can only be reused if another pod within same namespace uses same volume. What happens if another pod but in different namespace uses the volume? I am not super familiar with StorageOS and hence these questions.

Reply:
Yes, a volume can be used by multiple pods on the same node if they are in the same namespace as the PVC.

`b.podNamespace` looks to be incorrect there. Should be `b.volNamespace`. I'll change that.

The name variable name `globalPDPath` can be changed too. It does sounds misleading. It's just the target path where a loop device is mounted and made accessible to a pod.
I'll rename it to target path.

Reply:
okay, once tests are fixed I will lgtm.